src/pages/MainPage.vue

```diff
@@ -412,12 +436,13 @@ export default {
         agent: '',
         contents: newContents
       };
+      this.$store.commit('setActiveRoad', tempRoadID);
```

Comment:
is it a problem if this happens before the `setRoad` right below?

Reply:
I've found that all commits in a cycle run before it updates the DOM, so this should not be a problem.

Reply:
There shouldn't be a reason not to switch the order so we might as well put it in the order that seems safer